apps/researcher/src/app/[locale]/communities/[slug]/edit-community-form.tsx

```diff
@@ -16,25 +17,22 @@ interface Props {
   slug: string;
   description?: string;
   attributionId?: string;
+  licence?: string;
+  licenceToAccept: string;
```

Comment:
I don't fully understand the difference between these props - to me it suggests there can be a difference between a license and a license that can be accepted. Is that correct?

Reply:
I have removed `licenceToAccept` here, this is just the license in the environment variable that will be used to set the license if checked. But instead of passing the environment variable, I have imported it there where it is needed.